README.md

```diff
@@ -217,6 +220,7 @@ This plugin creates device plugin endpoints based on the configurations given in
 | "deviceType"     | N        | Device Type for a resource pool.                                                                                  | string value of supported types. Default: "netDevice" | Currently supported values: "accelerator", "netDevice"          |
 | "excludeTopology" | N       | Exclude advertising of device's NUMA topology          | bool Default: "false"    | "excludeTopology": true          |
 | "selectors"      | N        | A map of device selectors. The "deviceType" value determines the "selectors" options.                             | json object as string Default: null                   | Example: "selectors": {"vendors": ["8086"],"devices": ["154c"]} |
+| "allocatePolicy" | N        | Preferred device allocation policy for a resource pool. The "allocatePolicy" value determines which device in a resource pool is allocated first | string value of supported allocate policy. Default: "" | Currently supported values: "", "packed" |
```

Comment:
Am I assuming this correctly ? 
Packed : VFs are allocated under the same PF

Reply:
Yes, correct.